Gemfile

```diff
@@ -79,6 +79,7 @@ end
 
 group :development do
   gem 'hotwire-spark'
+  gem 'letter_opener' # Preview ActionMailer messages
```

Comment:
I can be convinced that popping up a window showing each formatted email in the browser as it would be sent is too heavyweight, in which case I'd opt for changing `delivery_method` in dev to `:file` and then documenting in the README where Rails stuffs those.

Reply:
The less heavyweight option seems preferable.

Reply:
I'll change it.